src/lib/translators/mongo.ts

```diff
@@ -8,6 +8,7 @@ import { $$, combinations, escapeForUseInRegExp } from '@/lib/helpers';
 import { OutputStep, StepMatcher } from '@/lib/matcher';
 import * as S from '@/lib/steps';
 import { BaseTranslator, ValidationError } from '@/lib/translators/base';
+import { ADVANCED_DATE_EXTRACT_MAP, DATE_EXTRACT_MAP } from '@/lib/translators/mongo_dates';
```

Comment:
Nitpick: our js/ts source files are generally named with dashes instead of underscores